src/runtime/auto-prompt-manager.ts

```diff
@@ -695,6 +712,15 @@ export class AutoPromptManager {
       );
     }
 
+    // ** Frequency Capping Impressions **
+    const enableFrequencyCappingLocalStorage = isExperimentOn(
+      this.doc_.getWin(),
```

Comment:
Isn't the flag set up as mendel flag propagated via article response? I think this function is for checking client-only flag

Reply:
Updated to check for article experiment.